src/Twilio/TwilioNoAuth.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Twilio.Clients;
+using Twilio.Exceptions;
+using Twilio.Clients.NoAuth;
+
+namespace Twilio
+{
+    /// <summary>
+    /// Default Twilio Client
+    /// </summary>
+    public class TwilioNoAuthClient
```

Comment:
Discussed over call, Athira will check if we remove this class, what is the downside as we are not taking input from customer.